cliquet_docs/api/resource.rst

```diff
@@ -355,9 +355,14 @@ The POST response body is a JSON mapping containing:
 - ``permissions``: *optional* a json dict containing the permissions for
   the requested resource.
 
-If the request header ``If-Match`` is provided, and if the record has
+If the request header ``If-Match: "<timestamp>"`` is provided as described in
+the :ref:`section about timestamps <server-timestamps>`, and if the collection has
 changed meanwhile, a ``412 Precondition failed`` error is returned.
```

Comment:
We should document the `details` property sent along the JSON body, useful to know about the conflicting record data.